components/ProjectList.js

```diff
@@ -17,49 +17,59 @@ export default function ProjectList({ projects, handleDeleteProject }) {
 
   return (
     <Wrapper>
-      <StyledList>
-        <SearchBar
-          type="text"
-          placeholder="search"
-          onChange={(event) => {
-            setSearchTerm(event.target.value);
-          }}
-        />
-        {projects
-          .filter((val) => {
-            if (searchTerm == "") {
-              return val;
-            } else if (
-              val.title.toLowerCase().includes(searchTerm.toLowerCase())
-            ) {
-              return val;
-            }
-          })
-          .map((project, index) => (
-            <ProjectItem key={index}>
-              <Link href={`/project/${project.id}`}>{project.title}</Link>
-              <DeleteButton
-                type="button"
-                onClick={() => setProjectToDelete(project.id)}
-              >
-                x
-              </DeleteButton>
-            </ProjectItem>
-          ))}
-      </StyledList>
-      <ModalDeleteProject
-        appearModalDeleteProject={projectToDelete}
-        handleDeleteProject={() => handleDeleteProject(projectToDelete)}
-        handleClose={() => {
-          setProjectToDelete(null);
+      <SearchBar
+        type="text"
+        placeholder="search"
+        onChange={(event) => {
+          setSearchTerm(event.target.value);
         }}
       />
+      <ProjectWrapper>
+        <StyledList>
+          {projects
+            .filter((val) => {
+              if (searchTerm == "") {
+                return val;
+              } else if (
+                val.title.toLowerCase().includes(searchTerm.toLowerCase())
+              ) {
+                return val;
+              }
+            })
```

Comment:
Nice! this filter is at the correct position. But instead of returning an object or undefined I would like you to return a boolean value inside the filter as it should be. Remember: if the item should be contained in the output Array you return a `true` and otherwise a `false`. You can add a final `else` condition where you return the `false` for example